l10n/en/mozilla-account-promo.ftl

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Variables
+#   $class (string) = CSS Class name for banner title
+moz-account-promo-title = One login. <br> <span { $class }>Everything</span> { -brand-name-mozilla }.
+
+# Variables
+#   $sign_in_url - link to
+#   $learn_more_url - link to https://www.mozilla.org/en-US/firefox/accounts/
+moz-account-already-have = Already have an account? <a { $sign_in_url }>Sign in</a> or <a { $learn_more_url }>learn more</a> about { -brand-name-mozilla }
```

Comment:
@reemhamz Hi Reem! 👋 Sorry to bother, but you maybe still remember if there was any discussion about this sentence or if the copy brief defined it this way? I have a hunch it was supposed to be: 

```suggestion
moz-account-already-have = Already have an account? <a { $sign_in_url }>Sign in</a> or <a { $learn_more_url }>learn more</a> about { -brand-name-mozilla-account }
```

I've opened #14460 for it, feel free to wontfix if the string is indeed supposed to be like this.